apps/studio/prisma/types.ts

```diff
@@ -9,13 +9,16 @@ import type {
   IsomerPageSchemaType as _IsomerPageSchemaType,
   IsomerSchema as _IsomerSchema,
   IsomerSiteConfigProps as _IsomerSiteConfigProps,
+  IsomerSiteThemeProps as _IsomerSiteThemeProps,
   IsomerSiteWideComponentsProps as _IsomerSiteWideComponentsProps,
 } from "@opengovsg/isomer-components"
 
 declare global {
   // eslint-disable-next-line @typescript-eslint/no-namespace
   namespace PrismaJson {
+    // TODO: Rename all with XXXYYYJson instead of XXXJsonYYY
```

Comment:
agree with this comment, wondering if we shuold just have a nested type like `site.config`?